dts/bindings/arm/nordic,nrf21540-gpio.yaml

```diff
@@ -1,8 +1,5 @@
-#
-# Copyright (c) 2020 Nordic Semiconductor
-#
-# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
-#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
This also is correcting something in the previous commit.  Probably there should just be one commit.